### ISSUE TITLE
Slow updateBlockAttributes on Safari 

### ISSUE THREAD

**Issue by jegbagus** [WordPress/gutenberg#29370]
## Description
Very slow updateBlockAttributes on Safari 14.0.3 (macOs big sur)

## Step-by-step reproduction instructions
1. loop child block from parent. 
2. do updateBlockAttributes of child attribute from parent.

## Expected behaviour
updateBlockAttributes should be happened instantly. this kind of behavior not happen on other browser (firefox, google chrome, microsoft edge)

## Actual behaviour
very slow update when updateBlockAttributes executed on safari. 

## Screenshots or screen recording (optional)
https://share.getcloudapp.com/RBu4rXDn

## Code snippet (optional)
```javascript
useEffect(() => {
        if (isSelected) {
            const rootId = getBlockRootClientId(clientId);
            const childs = getBlocks(rootId);

            childs.map(child => {
                if (child.clientId !== clientId) {
                    updateBlockAttributes(child.clientId, { active: false });
                }
            });

            setAttributes({ active: true });
        }
    }, [isSelected]);
```

## WordPress information
- WordPress version: 5.6.2
- Gutenberg version: 10.0.1
- Are all plugins except Gutenberg deactivated? No
- Are you using a default theme (e.g. Twenty Twenty-One)? Yes

## Device information
- Device: iMac
- Operating system: macOs Big Sur
- Browser: Safari 14.0.3

**Comment by ellatrix** [comment]
Is this fixed by #30995?

**Comment by ellatrix** [comment]
Should be fixed after #30995. Please reopen if that's not the case :)
